src/ompl/tools/lightning/src/Lightning.cpp

```diff
@@ -124,10 +124,12 @@ void ompl::tools::Lightning::setup()
             if (filePath_.empty())
             {
                 OMPL_ERROR("No file path has been specified, unable to load experience DB");
+                return; 
             }
             else
             {
                 experienceDB_->load(filePath_);  // load from file
+                return; 
```

Comment:
Shouldn't the function drop out of this conditional and set the `configured_` flag here? It appears to successfully load.